airflow/providers/standard/provider.yaml

```diff
@@ -42,6 +42,7 @@ operators:
     python-modules:
       - airflow.providers.standard.time.operators.datetime
       - airflow.providers.standard.time.operators.weekday
+      - airflow.providers.standard.core.operators.bash
```

Comment:
My 2c, I'm really not sure we need the time/core part of this. Seems better to just have

```
airflow.providers.standard.operators.weekday
airflow.providers.standard.operators.bash
```

Reply:
i am ok with suggestion. :)

Reply:
removed core/time package folders from standard provider.